### PR TITLE
ticdc: add upgrading section

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add TiCDC upgrading section.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4499 https://github.com/pingcap/docs-cn/pull/4503
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -71,6 +71,23 @@ The following are descriptions of options available in the `cdc server` command:
...
+
+```shell
+tiup update --self && \
+tiup update --all && \
+tiup cluster upgrade <cluster-name> v4.0.6
+```
+
+### Notes for upgrade
+
+* In TiCDC v4.0.2, changes were made to the `changefeed` configuration. See [Compatibility notes for the configuration file](/production-deployment-using-tiup.md#step-3-edit-the-initialization-configuration-file) for details.
```
Comment (yikeke) [inline ticdc/manage-ticdc.md:88]:
```suggestion
* The `changefeed` configuration has changed in TiCDC v4.0.2. See [Compatibility notes for the configuration file](/production-deployment-using-tiup.md#step-3-edit-the-initialization-configuration-file) for details.
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -71,6 +71,23 @@ The following are descriptions of options available in the `cdc server` command:
 - `cert`: The path of the certificate file used by TiCDC, in the PEM format (optional).
 - `key`: The path of the certificate key file used by TiCDC, in the PEM format (optional).
 
+## Upgrade TiCDC using TiUP
+
+This section introduces how to upgrade the TiCDC-enabled cluster using TiUP. In the following example, assume that you need to upgrade TiCDC and the cluster to v4.0.6.
```
Comment (yikeke) [inline ticdc/manage-ticdc.md:76]: Waiting on https://github.com/pingcap/docs-cn/pull/4503#pullrequestreview-488344750

Comment (TomShawn) [reply ticdc/manage-ticdc.md:76]:
```suggestion
This section introduces how to upgrade the TiCDC cluster using TiUP. In the following example, assume that you need to upgrade TiCDC and the entire TiDB cluster to v4.0.6.
```

Comment (yikeke) [reply ticdc/manage-ticdc.md:76]: LGTM

**Review by yikeke — APPROVED** [review]
LGTM

**Review by amyangfei — COMMENTED** [review]
LGTM

**Pull request merged**